easybuild/easyconfigs/p/pymatgen/pymatgen-4.1.1-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = "PythonPackage"
+
+name = 'pymatgen'
+version = '4.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/pymatgen'
+description = """Python Materials Genomics is a robust materials analysis code that defines core object
+ representations for structures and molecules with support for many electronic structure codes."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('spglib-python', '1.9.4.2', versionsuffix),
```

Comment:
@migueldiascosta hmm, just noticed something: existing `pymatgen` easyconfig list `spglib` (i.e. the C library) as a dependency, while you list (only) the Python wrappers as dependency, without having `spglib` listed as a dependency of `spglib-python`...

Are we missing something here?